packages/transport/src/modules/accounts/data/IdentityDeletionProcess.ts

```diff
@@ -0,0 +1,39 @@
+import { serialize, type, validate } from "@js-soft/ts-serval";
+import { nameof } from "ts-simple-nameof";
+import { CoreDate, CoreId, CoreSynchronizable } from "../../../core";
+import { CachedIdentityDeletionProcess, CachedIdentityDeletionProcessJSON } from "./CachedIdentityDeletionProcess";
+
+export interface IdentityDeletionProcessJSON {
+    id: string;
+    cache?: CachedIdentityDeletionProcessJSON;
+    cachedAt?: string;
+}
+
+export interface IIdentityDeletionProcess {
+    id: CoreId;
+    cache?: CachedIdentityDeletionProcess;
+    cachedAt?: CoreDate;
+}
+
+@type("IdentityDeletionProcess")
+export class IdentityDeletionProcess extends CoreSynchronizable implements IIdentityDeletionProcess {
+    public override readonly technicalProperties = [nameof<IdentityDeletionProcess>((r) => r.id)];
```

Comment:
I think the ID is synced automatically.

Reply:
Yea, but without any synced property, the initial data wallet entry will not be created. And therefor no sync at all will happen.

Reply:
It should create a cacheChange modification..?

Reply:
I see - it shouldn't. We aren't prepared for having no fields. But your workaround should be safe.